guide/src/class.md

```diff
@@ -716,6 +718,67 @@ impl MyClass {
 }
 ```
 
+## Free functions
```

Comment:
I wonder, should this be called something broader like "Accepting `#[pyclass]` values as function arguments"? (That's a bit of a meal to say, maybe there's a nicer way to phrase it.)

It might also be helpful to add this section to the "list of links" at the top of the page.